lib/components/Input/TextArea.module.scss

```diff
@@ -13,6 +13,9 @@ $line-height: 5*$grid-size;
         left: 0px;
         right: 0px;
         visibility: hidden;
+        white-space: pre-wrap;
+        overflow-wrap: break-word;
+        padding-right: 6*$grid-size; // accounts for scroll bar

```

Comment:
why this value?

Reply:
This is slightly larger than the scrollbar width because if the ghost text box doesn't account for these it won't have a long enough height to account for all of the text because it gets squished down more by the scroll bar in the real text area.

Reply:
Can you add a comment explaining this? So we make sure not to change this in the future